src/CalcViewModel/UnitConverterViewModel.cpp

```diff
@@ -988,7 +980,7 @@ void UnitConverterViewModel::OnPaste(String^ stringToPaste, ViewMode mode)
             }
 
             // Negate is only allowed if it's the first legal character, which is handled above.
-            if (NumbersAndOperatorsEnum::None != op && NumbersAndOperatorsEnum::Negate != op)
```

Comment:
Nice catch.  Did you find this through visual inspection, or with a tool?

Reply:
> Nice catch. Did you find this through visual inspection, or with a tool?

Thanks. Visual inspection is the most reliable tool :)